CONTRIBUTING_CODE.md

```diff
@@ -44,13 +44,20 @@ Once you've agreed on a design for your bugfix or new feature, development again
 #### Patching an older release
 
 If your contribution is intended for an older release, the change will need to be cherry-picked into the appropriate release branch after it has been reviewed
-and merged into master. Once your reviewer agrees the patch is valid for cherry-picking, perform the following steps to create the cherry-pick PR.
+and merged into master. **To make sure this happens, apply the `cherry-pick-candidate` label to the pull request or ask your reviewer to do so.**
+
+Once your reviewer agrees the patch is valid for cherry-picking, perform the following steps to create the cherry-pick PR.
 
 1. Check out the release branch corresponding to your target release, for example: `git fetch upstream; git checkout release-v2.5`.
-1. Create the cherry-pick branch, for example: `git checkout -b cherry-pick-pr12345`
+1. Create the cherry-pick branch based off of the target branch, for example: `git checkout -b cherry-pick-pr12345-v2.5`
```

Comment:
That example could be `git checkout -b cherry-pick-pr12345-v2.5 release-v2.5`, to further reify the `based off of the target branch` part.  WDYT?